Runner.php

```diff
@@ -93,8 +92,8 @@ public function run($files)
 
             $phpmd = new PHPMD();
 
-            if (isset($this->config[CFG]['file_extensions'])) {
-                $fileExtensions = $this->config[CFG]['file_extensions'];
+            if (isset($this->config[Runner::CFG]['file_extensions'])) {
```

Comment:
could this be `self:: CFG `? similar to how you did above?

Reply:
I'm not sure; I was thinking not because it wasn't a 'static' function. I'll look up the rules; self:: would be nice.